src/main/scala/io/iohk/ethereum/jsonrpc/JsonRpcController.scala

```diff
@@ -120,6 +120,8 @@ class JsonRpcController(
       handle[SendTransactionRequest, SendTransactionResponse](personalService.sendTransaction, req)
     case req @ JsonRpcRequest(_, "eth_call", _, _) =>
       handle[CallRequest, CallResponse](ethService.call, req)
+    case req @ JsonRpcRequest(_, "eth_estimateGas", _, _) =>
+      handle[CallRequest, EstimateGasResponse](ethService.estimateGas, req)
```

Comment:
I've been testing `eth_estimateGas` and could it be that in geth the block parameter is not configurable (as in `eth_call`)? It seems to be always `Pending` https://github.com/ethereumproject/go-ethereum/blob/master/eth/api.go#L792.